CHANGELOG.md

```diff
@@ -273,6 +280,7 @@
 ## [0.0.33] - 2019.02.14
 - Last release without a changelog!
 
+[0.3.1]: https://github.com/spacetx/starfish/releases/tag/0.3.0
```

Comment:
should this be 0.3.1 in the link?

Reply:
You are correct.
Thanks for catching that.